migrations/2024-07-31-063531_alter_customer_id_in_payouts/down.sql

```diff
@@ -0,0 +1,14 @@
+-- ALTER TABLE payouts
+-- ALTER COLUMN customer_id
+-- SET
+--     NOT NULL,
+-- ALTER COLUMN address_id
+-- SET
+--     NOT NULL;
+
+-- Below query will add the columns to the end, which would require diesel_models changes
+-- ALTER TABLE payout_attempt
+-- ADD COLUMN IF NOT EXISTS customer_id VARCHAR(64),
+-- ADD COLUMN IF NOT EXISTS address_id VARCHAR(64);
```

Comment:
You can uncomment these queries and remove the `SELECT 1;`.

> Below query will add the columns to the end, which would require diesel_models changes

As for this, these revert queries won't affect code that's already compiled and is running, it would only affect code that is yet to be compiled. These queries won't cause any runtime issues with Docker images that have been built either (as long as all the columns that the application reads from / writes to are present).